src/db_impl.cc

```diff
@@ -261,13 +261,8 @@ Status TitanDBImpl::Open(const std::vector<TitanCFDescriptor>& descs,
 
   // Initialize GC thread pool.
   if (!db_options_.disable_background_gc && db_options_.max_background_gc > 0) {
-    auto bottom_pri_threads_num =
-        env_->GetBackgroundThreads(Env::Priority::BOTTOM);
-    env_->IncBackgroundThreadsIfNeeded(
-        db_options_.max_background_gc + bottom_pri_threads_num,
-        Env::Priority::BOTTOM);
-    assert(env_->GetBackgroundThreads(Env::Priority::BOTTOM) ==
-           bottom_pri_threads_num + db_options_.max_background_gc);
+    env_->IncBackgroundThreadsIfNeeded(db_options_.max_background_gc,
```

Comment:
what if other works need more bottom threads

Reply:
then the other work should decide how many threads it needs (I agree this env API is awkward..)